docker-hub/audit-log.md

```diff
@@ -4,7 +4,11 @@ keywords: Team, organization, activity, log, audit, activities
 title: Audit log
 ---
 
-> The Audit log feature is available for users subscribed to a Team plan. [Upgrade](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"} your existing account to a Team account to start using the Audit log feature.
+{% include upgrade-cta.html
+  body="The Audit log feature is available for users subscribed to a Team plan. Upgrade your existing account to start using the Audit log feature."
```

Comment:
Users can't be on a Team plan, and this feels slightly repetitive. How about:

```suggestion
  body="The Audit log feature is available for organizations subscribed to a Team plan. Upgrade your existing plan to start tracking events across your organization."
```

Reply:
Updated